mantidproject/mantid
# Allow columns in sliceviewer peaks table to be sorted

**Description of work.**

Enables the peaks table in the sliceviewer to allow sorting by clicking on the column header.

Also contains a small tweak to the formatting to reduce the number of digits in each column.



**To test:**

* Use script in #28506 to produce an mdws & peaks workspace.
* Start the sliceviewer
* Overlay the peaks workspace
* Sort the table by any column and see it does the expected thing
* Select a peak and make sure the view zooms to the correct point

Fixes #28697 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
